src/components/icons/svg/angle-thin-left.react.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
+    <title>arrow-left</title>
```

Comment:
PS: the names here are still "wrong". Should we update them as well?

Reply:
Uff. Good one. No opinion 😐

Reply:
I went ahead and fixed the names: https://github.com/commercetools/ui-kit/pull/262/commits/f38e0124a19214bd2a286ea10214473406938ac6

I also noticed some inconsistency in the SVG structures. I already checked and notified the design team about it.